course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -213,6 +213,7 @@ def format_course_run_data(self, body, course=None):
             'enrollment_start': self.parse_date(body['enrollment_start']),
             'enrollment_end': self.parse_date(body['enrollment_end']),
             'hidden': body.get('hidden', False),
+            'invite_only': body.get('invitation_only', False),
```

Comment:
Since this is a new field so I would have liked its name same as the LMS field name `invitation_only` but I am not sure if someone in future tries to pull same LMS `invitation_only` field with same name `invitation_only` in course discovery (master branch of upstream) service then we will have conflicts due to the migration.
So lets keep its same as it is now.